### PR TITLE
Add settings sections to unified search

### PR DESCRIPTION
**Pull request by nickvergessen**
Bildschirmfoto von 2020-08-03 17-01-20

### CODE REVIEW

File: apps/settings/lib/Search/SectionResult.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Settings\Search;
+
+use OCP\Search\ASearchResultEntry;
+
+class SectionResult extends ASearchResultEntry {
```
Comment (nickvergessen) [inline apps/settings/lib/Search/SectionResult.php:28]: All those empty files start to bother me already. Should just make the `ASearchResultEntry` class non-abstract :/

**Comment by nickvergessen** [comment]
The icons are still black when using dark-theme.
The issue is that the generic filter in server does not react to unified-search items. However if it would, talk icons and profile pictures would also be inverted. This is a generic issue which needs fixing somehow.

File: apps/settings/lib/Search/SectionResult.php (hunk 1 above)
Comment (skjnldsv) [reply apps/settings/lib/Search/SectionResult.php:28]: @nickvergessen please do :)

**Review by skjnldsv — APPROVED** [review]

**Review by jancborchardt — APPROVED** [review]
Yeeeeah, really really cool! :) Finally a start of search in settings.

**Pull request merged**